utils/python/CIME/test_status.py

```diff
@@ -18,21 +18,23 @@
 # Special statuses that the overall test can be in
 TEST_DIFF_STATUS     = "DIFF"   # Implies a failure in one of the COMPARE phases
 NAMELIST_FAIL_STATUS = "NLFAIL" # Implies a failure in the NLCOMP phase
+TEST_BFAIL_STATUS    = "BFAIL"  # Implies baselines are missing in the COMPARE_baseline phase
 
 # The valid phases
-INITIAL_PHASE         = "INIT"
-CREATE_NEWCASE_PHASE  = "CREATE_NEWCASE"
-XML_PHASE             = "XML"
-SETUP_PHASE           = "SETUP"
-NAMELIST_PHASE        = "NLCOMP"
-SHAREDLIB_BUILD_PHASE = "SHAREDLIB_BUILD"
-MODEL_BUILD_PHASE     = "MODEL_BUILD"
-RUN_PHASE             = "RUN"
-THROUGHPUT_PHASE      = "TPUTCOMP"
-MEMCOMP_PHASE         = "MEMCOMP"
-MEMLEAK_PHASE         = "MEMLEAK"
-COMPARE_PHASE         = "COMPARE" # This is one special, real phase will be COMPARE_$WHAT
-GENERATE_PHASE        = "GENERATE"
+INITIAL_PHASE          = "INIT"
+CREATE_NEWCASE_PHASE   = "CREATE_NEWCASE"
+XML_PHASE              = "XML"
+SETUP_PHASE            = "SETUP"
+NAMELIST_PHASE         = "NLCOMP"
+SHAREDLIB_BUILD_PHASE  = "SHAREDLIB_BUILD"
+MODEL_BUILD_PHASE      = "MODEL_BUILD"
+RUN_PHASE              = "RUN"
+THROUGHPUT_PHASE       = "TPUTCOMP"
+MEMCOMP_PHASE          = "MEMCOMP"
+MEMLEAK_PHASE          = "MEMLEAK"
+COMPARE_PHASE          = "COMPARE" # This is one special, real phase will be COMPARE_$WHAT
+COMPARE_BASELINE_PHASE = "COMPARE_baseline"
```

Comment:
I like that you distinguish this as a separate phase, addressing the request in issue #396 . However, if you make this change, I'd request:

1. Changing this code in system_tests_common.py:

```python
            self._test_status.set_status("%s_baseline" % COMPARE_PHASE, status, comments=ts_comments)
```

Now this should use COMPARE_BASELINE_PHASE in place of "%s_baseline" % COMPARE_PHASE.

2. To make this phase more distinct from the in-test comparison, I'd suggest renaming this to something like BASELINE_COMPARE - i.e., so the name looks more different from the COMPARE_whatever used for the in-test comparison.